tests/integration/targets/xml/tasks/test-remove-attribute-nochange.yml

```
@@ -0,0 +1,28 @@
+---
+  - name: Setup test fixture
+    copy:
+      src: results/test-remove-attribute.xml
+      dest: /tmp/ansible-xml-beers.xml
+
+
+  - name: Remove non-existing '/business/rating/@subjective'
+    xml:
+      path: /tmp/ansible-xml-beers.xml
+      xpath: /business/rating/@subjective
+      state: absent
+    register: remove_attribute
+
+  - name: Compare to expected result
+    copy:
+      src: results/test-remove-attribute.xml
+      dest: /tmp/ansible-xml-beers.xml
+    check_mode: yes
+    diff: yes
+    register: comparison
+
+  - name: Test expected result
+    assert:
+      that:
+      - remove_attribute.changed == false
+      - comparison.changed == false  # identical
```

The standard Ansible test way would be:
```suggestion
      - remove_attribute is not changed
      - comparison is not changed  # identical
```
What you did is consistent with what the other tests for `xml` do though. Feel free to ignore this comment.

---

Yes, I choose consistency over modernization.